src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -282,6 +284,8 @@ struct CORINFO_InstructionSetFlags
             AddInstructionSet(InstructionSet_Sha1_Arm64);
         if (HasInstructionSet(InstructionSet_Sha256))
             AddInstructionSet(InstructionSet_Sha256_Arm64);
+        if (HasInstructionSet(InstructionSet_Sve))
```

Comment:
same here, want to understand difference between `InstructionSet_Sve` vs. `InstructionSet_Sve_Arm64` instead of `InstructionSet_Sve` and `InstructionSet_Sve2`.

Reply:
It doesn't look like this PR is adding `Sve2`

Reply:
sure, what I was trying to get is that since we have generated boiler plate code for encodings, etc. of both SVE and SVE2 instructions and we should define `SVE2` as well (instead of doing it  in future PR).

Reply:
There is a ton of work to do and we should really focus on finishing SVE before we start on Sve2, IMO

We don’t want to be in a scenario where we leave Sve2 incomplete and half done

Reply:
Alternative suggestion: We should focus first on the the instructions that give extra functionality that are not in NEON and will be useful for the existing SDK libraries. That may include some SVE2 instructions. Doing that would ensure some performance uplift for .NET9.

On my TODO list is to rewrite some of the SDK routines in SVE and see if I can find any performance improvements. Then use that to prioritise the SVE work.

Reply:
It would definitely be helpful to know what that list is.

Having an incomplete ISA is generally something we try to avoid as it can lead to issues when exposing APIs later, missed functionality, etc.